tests/integration/testlib/apply.py

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import libnmstate
+
+from libnmstate.schema import Descriptions
+
+
+def apply_with_description(description, desired_state, *args, **kwargs):
+    desired_state[Descriptions.KEY] = {"test-description": description}
```

Comment:
IMHO this should be considered a description of the state in imperative mood. Since a test might use several states, "test-description" is not accurate for a single state.

Reply:
Changed to a string describing the whole YAML.